src/couch_log/src/couch_log_config.erl

```diff
@@ -104,4 +105,23 @@ transform(strip_last_msg, "false") ->
     false;
 
 transform(strip_last_msg, _) ->
-    true.
+    true;
+
+transform(filter_fields, FieldsStr) ->
+    Default = [pid, registered_name, error_info, messages],
```

Comment:
Should this maybe go to a `DEF`?

Reply:
I followed the approach in other functions of this module (see line 100 and 104).